### PR TITLE
Deps upgrade, general cleanup

### PR DESCRIPTION
**Pull request by seanforyou23**
This PR upgrades all out of date dependencies, removes tslint, applies some formatting changes and tweaks the readme. No major changes, just keeping things fresh and up to date with latest PatternFly changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,12 +8,11 @@ Out of the box you'll get an app layout with chrome (header/sidebar), routing, b
...
 ## Quick-start
 ```bash
-npm install yarn -g # ensure you have yarn on your machine globally
 git clone https://github.com/patternfly/patternfly-react-seed # clone the project
 cd patternfly-react-seed # navigate into the project directory
-yarn # install patternfly-react-seed dependencies
-yarn build # build the project
-yarn start # start the development server
+npm install # install patternfly-react-seed dependencies
+npm run build # build the project
```
Comment (ibolton336) [inline README.md]: Just curious - why the switch back to mom from yarn?

Comment (seanforyou23) [reply README.md]: Both still work, I just changed the Quick start section to reference npm so it's one less step for for new users. Hoping that people see the yarn commands listed below and the yarn.lock to realize both work. Do you think it's confusing? I can instead add two sections to show how to start with each?

Comment (ibolton336) [reply README.md]: IMO i would just stick with one. One other thing that stuck out on this PR was adding the lockfile to gitignore. Just went through this on a project and we ended up checking in the lockfile so it keeps everyones deps in sync.

Comment (seanforyou23) [reply README.md]: It's a good point! Yea with a little more thought it seems we should just pick one. I think going back to npm is the way to go - it's simpler to start, and there's nothing stopping a consumer from switching over to yarn if they like. I'll send some updates shortly!

**Comment by seanforyou23** [comment]
@cdcabrera left in the `format` script so you can manually run prettier so it's still in there. Just the tsconfig setting that we removed. This made me realize I had dropped axe so I've added that back. I think we're gonna do some more updates to the readme format soon so we can definitely revisit. Thanks!!

**Pull request merged**